contrib/docker/build.sh

```diff
@@ -9,8 +9,8 @@ set -e
 echo "creating debian image..."
 docker run --rm -it -v `pwd`/../..:/build ebusd-buildenv ./make_debian.sh
 export EBUSD_VERSION=`cat ../../VERSION`
-export EBUSD_ARCH=`docker version|grep -i "Arch[^:]*server"|head -n 1|sed -e 's#^.*/##'`
+export EBUSD_ARCH=`docker version|grep -i "Arch:"|head -n 1|sed -e 's#^.*/##'`
```

Comment:
this is going to pick the client version instead of the server version (at least on debian), which is not correct. how does the docker version output look in your environment?

Reply:
It‘s in #148

Reply:
okay, then I guess the best way to pick the server version is usind tail instead of head. should work in both environments